src/components/side-panel/side-panel.component.ts

```diff
@@ -115,8 +118,11 @@ export class SidePanelComponent implements OnInit, OnDestroy {
                 : SidePanelAnimationState.Closed;
 
             // only if the open state changed should we emit the latest value
-            this.openChange.emit(isOpen);
+            if (!this._isInitial) {
```

Comment:
Instead of having a variable to store if it is the initial emission instead could we have a second subscription to `open$` for handling the emission:

```typescript
this.service.open$.pipe(skip(1), distinctUntilChanged(), takeUntil(this._onDestroy))
    .subscribe(isOpen => this.openChange.emit(isOpen));
```

Reply:
updated